.github/lighthouse/budget.json

```diff
@@ -0,0 +1,49 @@
+[
+  {
```

Comment:
@vickywane why do we need a lighthouse config for serverless functions? LH is for PWA apps.

Reply:
I'll remove it